### PR TITLE
Use docker compose for cassandra integration tests

### PR DESCRIPTION
**Pull request by hellspawn679**
## Which problem is this PR solving?
- https://github.com/jaegertracing/jaeger/issues/5485

## Description of the changes
- updated `cassaandra-integration-test.sh` to use docker compose 
- update dependabot

## How was this change tested?
- `bash scripts/cassandra-integration-test.sh 3 v003 v1`    ` bash scripts/cassandra-integration-test.sh 3 v003 v2 ` bash `scripts/cassandra-integration-test.sh 4 v004 v1` `scripts/cassandra-integration-test.sh 4 v004 v2`
 
## Checklist
- [ ] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [ ] I have signed all commits
- [ ] I have added unit tests for the new functionality
- [ ] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `yarn lint` and `yarn test`

### CODE REVIEW

File: scripts/cassandra-integration-test.sh [hunk 1]
```diff
@@ -53,13 +46,14 @@ apply_schema() {
 
 run_integration_test() {
   local version=$1
+  local major_version=${version%%.*}
```
Comment (yurishkuro) [inline scripts/cassandra-integration-test.sh:49]: Can we do the same with es/os scripts? Their matrix was changed "7.x" -> "7", which makes the workflow names look weird. I would prefer to go back to 7.x / 8.x

Comment (yurishkuro) [reply scripts/cassandra-integration-test.sh:49]: Can be a separate PR for that.

File: scripts/cassandra-integration-test.sh [hunk 2]
```diff
@@ -15,22 +15,15 @@ check_arg() {
...
-  local params=(
-    --detach
-    --publish 9042:9042
-    --publish 9160:9160
-  )
-  local cid
-  cid=$(docker run "${params[@]}" "${image}:${tag}")
-  echo "cid=${cid}" >> "$GITHUB_OUTPUT"
-  echo "${cid}"
+  local major_version=$1
```
Comment (yurishkuro) [inline scripts/cassandra-integration-test.sh:18]: why is this needed?

File: scripts/cassandra-integration-test.sh [hunk 3]
```diff
@@ -53,13 +46,14 @@ apply_schema() {
 
 run_integration_test() {
   local version=$1
+  local major_version=${version%%.*}
   local schema_version=$2
   local jaegerVersion=$3
   local primaryKeyspace="jaeger_v1_dc1"
   local archiveKeyspace="jaeger_v1_dc1_archive"
+  local compose_file="docker-compose/cassandra/v3.yaml"
```
Comment (yurishkuro) [inline scripts/cassandra-integration-test.sh:54]: why is this hardcoded to v4?

File: scripts/cassandra-integration-test.sh (hunk 1 above)
Comment (hellspawn679) [reply scripts/cassandra-integration-test.sh:48]: ok i will create a new pr after this

**Review by yurishkuro — APPROVED** [review]
Thanks!

**Comment by yurishkuro** [comment]
> Signed-off-by: Your Name <you@example.com>

I would advise you to fix this setting in your git.

File: .github/dependabot.yml [hunk 4]
```diff
@@ -136,3 +136,10 @@ updates:
     allow:
       - dependency-name: "bitnami/kafka"
         update-types: ["version-update:semver-minor"]
+  - package-ecosystem: docker
+    directory: /docker-compose/cassandra
```
Comment (yurishkuro) [inline .github/dependabot.yml:140]: @hellspawn679 I don't think this is working, the dependabot runs did not pick this up. I suspect that when you pick `docker` ecosystem and a directory, dependabot looks for standard file names like `Dockerfile` or `docker-compose.yml`. Since you're using custom file names like `v3.yml`, dependabot doesn't know that they are for docker-compose. We probably need to list the files explicitly instead of specifying a dir only.

Comment (yurishkuro) [reply .github/dependabot.yml:140]: https://github.com/jaegertracing/jaeger/issues/5552
